<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="MineSizeBox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="MinSizeLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="MaxSizeLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>90</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MaxSizeBox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>110</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ContainingBox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>190</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ContainingLabel">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>140</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Containing - Space seperated</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GoButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>510</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>GO!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PIDBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PIDLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>PID or program name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SubstitutionsBox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>260</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="SubstitutionLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>230</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Substitutions</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="OutputBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>781</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="MaxHeapObjBox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="MaxHeapSizeLabel">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>290</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Max heap object size</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SearchStackCheckBox">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>370</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Stack</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SearchBssCheckBox">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>390</y>
      <width>88</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Bss</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SearchBssCheckBox_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>410</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Unique Size(TODO)</string>
    </property>
   </widget>
   <widget class="QLabel" name="BackendErrorLabel">
    <property name="geometry">
     <rect>
      <x>207</x>
      <y>7</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
